GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -290,9 +299,15 @@ public void Add(GitRevision revision, RevisionNodeFlags types = RevisionNodeFlag
         {
             _revisionGraph.Add(revision, types);
 
-            if (ToBeSelectedObjectIds.Remove(revision.ObjectId))
+            if (ToBeSelectedObjectIds.Contains(revision.ObjectId))
             {
-                ToBeSelectedRowIndexes.Enqueue(_revisionGraph.Count - 1);
+                var rowIndexToBeSelectedWithOrderInfo = new RowIndexToBeSelectedWithOrderInfo
+                {
+                    RowIndex = _revisionGraph.Count - 1,
+                    SelectionOrder = ToBeSelectedObjectIds.IndexOf(revision.ObjectId)
```

Comment:
```csharp
UserControls\RevisionGrid\RevisionDataGridView.cs(307,68):
 error CS1503: Argument 2:
 cannot convert from 'GitUIPluginInterfaces.ObjectId' to 'System.Func<GitUIPluginInterfaces.ObjectId, bool>' 
```

Reply:
See my review comment about the compile error introduced by the latest commit.